src/style-stars--4.css

```diff
@@ -0,0 +1,3 @@
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
The CSS selector targets elements with a class of `stars__star` that are children of an element with a class of `stars--4`. However, the checklist advises against styling using tag names and recommends using classes instead. If `stars__star` is not a class but a tag, consider changing it to a class for consistent styling and to avoid potential conflicts with other CSS rules.